teku/src/main/java/tech/pegasys/teku/TekuFacade.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku;
+
+import tech.pegasys.teku.config.TekuConfiguration;
+
+/**
+ * The main entry point to start Teku node
+ *
+ * <p>CAUTION: this API is unstable and primarily intended for debugging and testing purposes this
+ * API might be changed in any version in backward incompatible way
+ */
+public interface TekuFacade {
+
+  static TekuFacade getInstance() {
+    return Teku.INSTANCE;
+  }
```

Comment:
I'd make this a `static TekuFacade create()` that creates a new `Teku` instance rather than using the static.

Reply:
That actually could be a bit confusing: creating several `Teku` instances and each of them can create several `Node` instances 🤔

Reply:
@ajsutton could you please take another look: c5ef2a1

Reply:
I like the look of those static methods yeah. Think it's good to go.